associated-token-account/program/src/instruction.rs

```diff
@@ -26,10 +26,25 @@ pub enum AssociatedTokenAccountInstruction {
 pub fn create_associated_token_account(
     funding_address: &Pubkey,
     wallet_address: &Pubkey,
-    spl_token_mint_address: &Pubkey,
+    token_mint_address: &Pubkey,
+) -> Instruction {
+    create_associated_token_account_with_program_id(
+        funding_address,
+        wallet_address,
+        token_mint_address,
+        &spl_token::id(),
+    )
+}
+
+/// Creates CreateAssociatedTokenAccount instruction
+pub fn create_associated_token_account_with_program_id(
```

Comment:
Or should we just yank ATA 1.0.4 and publish a 1.0.5 with the program_id folded into `create_associated_token_account()`?  1.04 is just 3 days old with probably not a lot of adoption yet

Reply:
Yeah, maybe we should. @joncinque and I were pondering making that yank anyway, due to: https://github.com/solana-labs/solana-program-library/issues/2791